cmd/image/import/import.go

```diff
@@ -171,9 +175,14 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --object rhel-83-1003
 		if err != nil {
 			return err
 		}
+		//By default Bucket Access is private
+		BucketAccess := "private"
```

Comment:
I don't see a reason why this needs to be exported!

```suggestion
		bucketAccess := "private"
```

Reply:
Done.